code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -72,11 +71,15 @@
 	return ..()
 
 /obj/machinery/chem_dispenser/process()
-	if(recharged < 0)
```

Comment:
Shouldn't this be a timer instead?

Reply:
we already have process(), we don't need a timer